src/R/Editor/Application.Test/Fixtures/REditorApplicationServicesFixture.cs

```diff
@@ -1,19 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System.Collections.Generic;
-using System.Diagnostics.CodeAnalysis;
-using System.Linq;
-using Microsoft.Markdown.Editor.Test;
-using Microsoft.R.Editor.Test.Fixtures;
-
-namespace Microsoft.R.Editor.Application.Test.Fixtures {
-    // Fixture doesn't import itself. Use AssemblyFixtureImportAttribute
-    [ExcludeFromCodeCoverage]
-    public class REditorApplicationServicesFixture : MarkdownEditorServicesFixture {
```

Comment:
Not sure why this is different, should be same as master... R.Support was removed back in my previous merge (large editor split PR)